core/src/main/java/com/backblaze/b2/client/structures/B2Bucket.java

```diff
@@ -97,6 +104,7 @@ public String toString() {
                 (bucketInfo == null ? 0 : bucketInfo.size()) + " infos," +
                 (corsRules == null ? 0 : corsRules.size()) + " corsRules," +
                 (lifecycleRules == null ? 0 : lifecycleRules.size()) + " lifecycleRules," +
+                (options == null ? 0 : options.size()) + " options," +
```

Comment:
Wondering about listing the options here.  Would that be reasonable

Reply:
updated